src/main/kotlin/com/spectralogic/bp/bench/cli/WriteToTapeCommand.kt

```diff
@@ -62,11 +62,15 @@ class WriteToTapeCommand :
         val client = Ds3ClientHelpers.wrap(
             Ds3ClientBuilder.create(endpoint, Credentials(clientId, secretKey))
                 .withHttps(false)
+                .withBufferSize(bufferSize)
                 .build()
         )
         client.ensureBucketExistsByName(bucket, dataPolicy)
-        val job = client.startWriteJob(bucket, ds3ObjectSequence().toList(), WriteJobOptions.create().withPriority(Priority.URGENT))
-        job.transfer { PositionableReadOnlySeekableByteChannel(Channels.newChannel(AZInputStream())) }
+        val job = client.startWriteJob(bucket, ds3ObjectSequence().toList(), WriteJobOptions.create()
+            .withPriority(Priority.URGENT))
```

Comment:
Should we be defaulting this to `URGENT`?

Reply:
What should it be? I was figuring a benchmark would want to preempt?

Reply:
Probably normal, but it might be a good idea to allow the user to override this setting via a command line arg.